### ISSUE TITLE
Sessions Table Animation Makes App feel Sluggish

### ISSUE THREAD

**Issue by xxxoooxoxo** [humanlayer/humanlayer#590]
### Problem to solve / Expected Behavior

On the sessions overview, the animation when navigating with J/K feels slugish. The animation should be fast/not applied with keyboard nav/ only apply to the icon and not the row border

### Proposed solution

Make the animation speed faster, and only apply to the select box, not the row body color

**Comment by dexhorthy** [comment]
Not sure if this is animated, might just be a rendering lag or separate event. I think this might be around paging / too many items in the dom

**Comment by dexhorthy** [comment]
Lets use tanstack virtual and see if this improves the rendering perf
